.github/workflows/build-all-arduino.yml

```
@@ -0,0 +1,91 @@
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build:
+    needs: create-tag
+    runs-on: ubuntu-latest
+    timeout-minutes: 30  # Increased timeout for large downloads
+
+    steps:
+    - name: 📦 Checkout code
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ needs.create-tag.outputs.tag_name }}
```

actionlint (1.7.7)

property "tag_name" is not defined in object type {}

(expression)